docs/products/postgresql/howto/identify-and-fix-pg-slow-queries.rst

```diff
@@ -0,0 +1,123 @@
+Identify and fix PostgreSQL® slow queries 
+===========================
```

Comment:
The bar under a title should be longer than the title itself, otherwise Vale might complain

```suggestion
===================================================
```